core/src/main/java/org/apache/carbondata/core/datastore/page/ColumnPage.java

```diff
@@ -485,6 +496,11 @@ private void putNull(int rowId) {
   public abstract int getShortInt(int rowId);
 
   /**
+   * Get boolean value at rowId
+   */
+  public abstract boolean getBoolean(int rowId);
```

Comment:
The same as above, you can implement this function here instead of abstract

Reply:
ok, I have changed it